### PR TITLE
chore: canvas cleanup betterer styles object notation edition

### PR DESCRIPTION
**Pull request by nmarrs**
While working on canvas button element, noticed that we had a ton of betterer complaints about how canvas styles are notated

Combed through and switched over vast majority of style object complaints for canvas

### CODE REVIEW

File: public/app/features/canvas/elements/ellipse.tsx [hunk 1]
```diff
@@ -23,26 +22,26 @@ class EllipseDisplay extends PureComponent<CanvasElementProps<EllipseConfig, Ell
...
+  container: css({
+    display: 'table',
+    position: 'absolute',
+    top: '50%',
+    transform: 'translateY(-50%)',
+    width: '100%',
+    height: '100%',
+    backgroundColor: data?.backgroundColor,
+    border: `${data?.width}px solid ${data?.borderColor}`,
+    borderRadius: theme.shape.radius.circle,
```
Comment (nmarrs) [inline public/app/features/canvas/elements/ellipse.tsx:35]: this is set to 100% but still works the same as 50% value 🤷‍♂️

File: public/app/plugins/panel/canvas/components/connections/ConnectionAnchors.tsx [hunk 2]
```diff
@@ -114,36 +114,35 @@ export const ConnectionAnchors = ({ setRef, handleMouseLeave }: Props) => {
...
+    height: 'calc(100% + 60px)',
+  }),
+  anchor: css({
+    padding: `${ANCHOR_PADDING}px`,
+    position: 'absolute',
+    cursor: 'cursor',
+    width: `calc(5px + 2 * ${ANCHOR_PADDING}px)`,
+    height: `calc(5px + 2 * ${ANCHOR_PADDING}px)`,
+    zIndex: 100,
+    pointerEvents: 'auto',
```
Comment (nmarrs) [inline public/app/plugins/panel/canvas/components/connections/ConnectionAnchors.tsx:134]: losing the `!important` here didn't seem to impact the functionality of connection anchors 🤷‍♂️  side note I'm not sure it's possible to include `!important` notation via the css object notation

File: public/app/plugins/panel/canvas/editor/element/ConstraintSelectionBox.tsx [hunk 3]
```diff
@@ -141,48 +141,84 @@ const getStyles = (currentConstraints: Constraint) => (theme: GrafanaTheme2) =>
...
-    topConstraint: css`
-      ${currentConstraints.vertical === VerticalConstraint.Top ||
+    constraintHover: css({
+      '&:hover': {
+        fill: HOVER_COLOR,
+        fillOpacity: HOVER_OPACITY,
+      },
+    }),
+    topConstraint: css({
+      ...(currentConstraints.vertical === VerticalConstraint.Top ||
```
Comment (nmarrs) [inline public/app/plugins/panel/canvas/editor/element/ConstraintSelectionBox.tsx:151]: 😬

File: public/app/features/canvas/elements/ellipse.tsx (hunk 1 above)
Comment (drew08t) [reply public/app/features/canvas/elements/ellipse.tsx:35]: @Develer

File: public/app/plugins/panel/canvas/components/connections/ConnectionAnchors.tsx (hunk 2 above)
Comment (adela-almasan) [reply public/app/plugins/panel/canvas/components/connections/ConnectionAnchors.tsx:134]: I've seen it being used in code, but it didn't work for me.. so not sure if it actually works 😬

File: public/app/features/canvas/elements/ellipse.tsx (hunk 1 above)
Comment (Develer) [reply public/app/features/canvas/elements/ellipse.tsx:35]: @nmarrs with 50% height element doesn't match the container border.
See the comparison below.
Screenshot 2023-10-12 at 19 27 45
Screenshot 2023-10-12 at 19 27 25

**Review by adela-almasan — APPROVED** [review]
⭐

**Pull request merged**